src/GraphErrorHandler.ts

```diff
@@ -58,6 +58,12 @@ export class GraphErrorHandler {
 	 *  }
 	 */
 	private static constructErrorFromResponse(error: any, statusCode: number): GraphError {
+		if (!error.error) {
```

Comment:
values of error being `undefined` or `null` are not handled.

Reply:
Is `error` guaranteed to be a valid object?

Reply:
This function is called if error.error exists.